### ISSUE TITLE
Can't start app on OSX

### ISSUE THREAD

**Issue by judodan** [federkasten/appbundle-maven-plugin#39]
This is my first time trying this.  It would save me an immense amount of time if I could get it working.  I followed the instructions in the README and it did, in fact, create the .app file!

Unfortunately, when I try to start the app, the icon shows in the command bar, then disappears.  Can't seem to get it to start.

**Comment by federkasten** [comment]
I think your application halted with some errors.
Please check `mainClass` field in `pom.xml` is correct or refer to your application logs.

**Comment by judodan** [comment]
The class looks fine.  It's a swing app if that make a difference.  After executing, the following is in system.log:

Aug  8 08:29:39 MacBook-Pro-6 lsd[712]: LaunchServices: Could not store lsd-identifiers file at /private/var/db/lsd/com.apple.lsdschemes.plist
Aug  8 08:29:40 MacBook-Pro-6 launchservicesd[87]: SecTaskLoadEntitlements failed error=22
Aug  8 08:29:40 --- last message repeated 1 time ---
Aug  8 08:29:40 MacBook-Pro-6 loginwindow[103]: -[SFLListManager(ServiceReplyProtocol) notifyChanges:toListWithIdentifier:] Notified of item changes to list with identifier com.apple.LSSharedFileList.RecentApplications
Aug  8 08:29:40 MacBook-Pro-6 appleeventsd[56]: SecTaskLoadEntitlements failed error=22

**Comment by judodan** [comment]
I finally found the issue.  While it wasn't this plugin, I want to document here (to help the next guy).  Turns out the problem was an issue with IntelliJ (my IDE), which uses a special compiler to build the UIs that are generated from the form designer.  Maven doesn't know about them, so you only get a partial build when you use Maven to build/package.  In order to be complete, I was doing a clean, then package.  I then figured out a workaround, which is to use IntelliJ Rebuild command, then the Maven package (without clean).  The resulting APP file appears to work!

I did do some searches how to automate that, but have not found a suitable solution.  Hope this helps somebody.
